ckanext/xloader/parser.py

```diff
@@ -31,6 +31,8 @@ def convert_types(self, extended_rows):
             for cell_index, cell_value in enumerate(row):
                 if cell_value is None:
                     row[cell_index] = ''
+                cell_value = str(cell_value).strip()  # strip white space around cell values
```

Comment:
The logic after this point assumes that `cell_value` might not be a string (in which case it is preserved unchanged). Converting all values to strings is undesirable.

Reply:
@ThrawnCA thanks! Yeah was trying to figure out how to strip and get the tests working. I will condition on str instance and that should hopefully work better.